andela/pippin-ah-backend
# #163053810 Create PR template

#### What does this PR do?
This PR creates a Pull Request template for the repo

#### Description of Task to be completed?
- Create a PULL_REQUEST_TEMPLATE file in a .github folder and then add the default pull request template for this project

#### How should this be manually tested?
- Create a new pull request from a branch
- The default Pull Request template should already be present in the 'Write' section.

### What are the relevant pivotal tracker stories?
#163053810 https://www.pivotaltracker.com/n/projects/2229853/stories/163053810/

### Screenshots (if appropriate)
None

#### Any background context you want to provide?
None
